### PR TITLE
format ons import date in clean cqc loc script

### PR DESCRIPTION
**Pull request by james-francis-MT**
# Description
Format ons import date to be used for the join

# How to test
Unit tests pass. Can be tested fully here: https://trello.com/c/xVrWiYbf/59-epic-4-add-columns-with-current-geographies-must

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: jobs/clean_cqc_location_data.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
 DATE_COLUMN_IDENTIFIER = "registration_date"
 
+ONS_FORMATTED_IMPORT_DATE_COL = "ons_postcode_import_date"
```
Comment (garypolzin) [inline jobs/clean_cqc_location_data.py:33]: Can you add this col name to utils>column_names>raw_data_files>ons_columns
so it matches the format of row 51 (should be fine in 'raw' given we don't have a cleaning script)

**Review by garypolzin — CHANGES_REQUESTED** [review]
Might be worth adding a test in main to assert that this function is called twice like we do with the other utils function

File: jobs/clean_cqc_location_data.py (hunk 1 above)
Comment (james-francis-MT) [reply jobs/clean_cqc_location_data.py:33]: Thought it felt strange being there as it isn't actually a column in the raw data but I can add it

Comment (garypolzin) [reply jobs/clean_cqc_location_data.py:33]: yea fair point, it's easy to change later so leave it here then

**Review by garypolzin — APPROVED** [review]

**Pull request merged**